cmd/root.go

```diff
@@ -59,6 +59,13 @@ Produces:
 NOTE the minimum supported SAST version is 9.3. SAST installations below this version should be upgraded in order to run this export tool. 
 `,
 	Run: func(cmd *cobra.Command, args []string) {
+
+		// Validate simIDVersion if provided
+		if simIDVersion < 0 || simIDVersion > 2 {
+			fmt.Println("Error: simIDVersion must be 0 (Default), 1 (Trim leading spaces), or 2 (Remove all spaces).")
+			os.Exit(1)
```

Comment:
Instead, please log the error and exit with panic, like is done near the end of this function:
```go
		exportErr := internal.RunExport(&allArgs)
		if exportErr != nil {
			log.Error().Err(exportErr)
			panic(exportErr)
		}
```

You can create an error with `fmt.Errorf(errorMessageString)`